src/components/Editor/Footer.js

```diff
@@ -152,6 +156,22 @@ class SourceFooter extends PureComponent<Props> {
     );
   }
 
+  renderSourceSummary() {
+    const { selectedSource, mappedSource } = this.props;
+    const source = selectedSource.toJS();
+    if (isOriginalId(source.id)) {
```

Comment:
we should just be able to check the `mappedSource` i.e
```js
if(mappedSource){
```

Reply:
fixed

Reply:
have you push your fixes yet? i can't see them yet